Bugfix/6750 name clash between npm request and parameter called request

pgrm opened the pull request:
### PR checklist

- [x] Read the [contribution guidelines](https://github.com/swagger-api/swagger-codegen/blob/master/CONTRIBUTING.md).
- [x] Ran the shell script under `./bin/` to update Petstore sample so that CIs can verify the change. (For instance, only need to run `./bin/{LANG}-petstore.sh` and `./bin/security/{LANG}-petstore.sh` if updating the {LANG} (e.g. php, ruby, python, etc) code generator or {LANG} client's mustache templates). Windows batch files can be found in `.\bin\windows\`.
- [x] Filed the PR against the correct branch: `3.0.0` branch for changes related to OpenAPI spec 3.0. Default: `master`.
- [x] Copied the [technical committee](https://github.com/swagger-api/swagger-codegen/#swagger-codegen-technical-committee) to review the pull request if your PR is targeting a particular programming langauge.

### Description of the PR

fixes https://github.com/swagger-api/swagger-codegen/issues/6750

rename `import request = require('request');` to `import $request = require('request');` to fix the clash described in #6750

Could someone of you please have a look @TiFu @taxpon @sebastianhaas @kenisteward @Vrolijkx

wing328 commented:
@pgrm thanks for the PR. For other generators, we add a prefix (e.g. localVar, local_var_) to the local variables. I wonder we can do something similar for TS node (request => localVarRequest)

We just did somethings similar for TS Fetch: https://github.com/swagger-api/swagger-codegen/pull/6717

pgrm commented:
@wing328 thx for the hint, I've renamed it to `localVarRequest` and I also renamed some other variables, in a separate commit though, so if you think shouldn't be done, I can undo it again quickly.

wing328 commented:
@pgrm the change looks good to me. Thanks for your contribution.
